gcn_package/models/gcn.py

```diff
@@ -48,41 +48,132 @@ def forward(self, x):
         x = self.classifier(x)
         return x
 
-class LoicGCN(torch.nn.Module):
-    def __init__(self, edge_index, edge_weight, n_timepoints=50, n_classes=2):
+class ChebConvBlock(torch.nn.Module):
```

Comment:
Do we want to make this private?

Reply:
wdym ? I don't care personally.

Reply:
Oh sorry okay I see. I don't think python has concept of private classes, how would you do it?

Reply:
Actually I am doing standalone tests for these blocks, so making them private will complexify testing.
 https://github.com/SIMEXP/gcn_package/blob/6dbd9df9f7a44728d0563789fa519767ddeeda47/gcn_package/tests/test_models.py#L22-L44